lib/contributionreward.js

```diff
@@ -93,19 +102,30 @@ class ContributionReward extends ExtendTruffleContract(SolidityContributionRewar
     const ethReward = web3.toBigNumber(options.ethReward);
     const externalTokenReward = web3.toBigNumber(options.externalTokenReward);
 
-    if ((nativeTokenReward < 0) || (reputationReward < 0) || (ethReward < 0) || (externalTokenReward < 0))
-    {
+    if (
+      nativeTokenReward < 0 ||
```

Comment:
Does this parameters can be type as unsigned somehow? so the is negative check can be avoided?

Reply:
This PR is about formatting, that is all. I would prefer to take note of other comments and address them elsewhere.